srcpkgs/suricata/template

```diff
@@ -1,17 +1,22 @@
 # Template file for 'suricata'
 pkgname=suricata
 version=4.1.2
-revision=1
+revision=2
 build_style=gnu-configure
-configure_args="--disable-gccmarch-native"
+configure_args="--disable-gccmarch-native $(vopt_enable rust_support rust) $(vopt_enable luajit)"
 hostmakedepends="pkg-config"
-makedepends="libnet-devel libpcap-devel pcre-devel libyaml-devel libcap-ng-devel file-devel"
+makedepends="libnet-devel libpcap-devel pcre-devel libyaml-devel libcap-ng-devel liblz4-devel file-devel jansson-devel nss-devel"
 short_desc="Next Generation Intrusion Detection and Prevention Engine"
 maintainer="Andrea Brancaleoni <abc@pompel.me>"
 license="GPL-2"
 homepage="https://suricata-ids.org/"
 distfiles="https://www.openinfosecfoundation.org/download/$pkgname-$version.tar.gz"
 checksum=73575b041a50cc48a2a53f6503ab4d355166d7acbd4997cd04045f848f8bea96
+makedepends+=" $(vopt_if luajit LuaJIT-devel) $(vopt_if rust_support rust) $(vopt_if rust_support cargo)"
```

Comment:
I would prefer that this is added to `makedepends` directly instead of adding `makedepends+=`.

For cross compilation: `cargo` should be in `hostmakedepends` and `build_helper="rust"` should be added to help setting up the cargo environment to allow cross compiling.